sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -65,7 +67,7 @@ public TextAnalyticsClient(Uri endpoint, TokenCredential credential, TextAnalyti
             _options = options;
 
             var pipeline = HttpPipelineBuilder.Build(options, new BearerTokenAuthenticationPolicy(credential, defaultScope));
-            _serviceRestClient = new TextAnalyticsRestClient(_clientDiagnostics, pipeline, endpoint.AbsoluteUri, TextAnalyticsClientOptions.GetVersionString(options.Version));
+            _serviceRestClient = new AnalyzeTextRestClient(_clientDiagnostics, pipeline, endpoint.AbsoluteUri, TextAnalyticsClientOptions.GetVersionString(options.Version));
```

Comment:
we need to instantiate `_cognitiveRestClient` too otherwise the calls are not going to get executed

Reply:
actually, I am unclear what the difference is between the two. Could you explain when to use which?